python/repositories.bzl

```diff
@@ -27,89 +27,79 @@ def py_repositories():
 ########
 # Remaining content of the file is only used to support toolchains.
 ########
-_DOC = "Fetch external tools needed for python toolchain."
-_ATTRS = {
-    "integrity": attr.string(mandatory = True),
-    "platform": attr.string(mandatory = True, values = PLATFORMS.keys()),
-    "python_version": attr.string(mandatory = True, values = TOOL_VERSIONS.keys() + MINOR_MAPPING.keys()),
-}
 
 def _python_repository_impl(rctx):
-    python_version = rctx.attr.python_version
     platform = rctx.attr.platform
-    release_filename = "cpython-{version}-{platform}-pgo+lto-{release_date}.tar.zst".format(
+    is_host_windows = "win" in rctx.os.name
+    is_target_windows = "windows" in rctx.attr.platform
+    python_version = rctx.attr.python_version
+    build = "static-install_only" if is_target_windows else "install_only"
```

Comment:
I'm not a Windows user. Would a static binary for Windows have the same limitation as a fully static binary for Linux that a shared object (i.e. a native Python extension) cannot be loaded at runtime (not able to do dlopen)?

Reply:
I'm not sure myself, but this was the only artifact that worked. My thought was to hopefully see something merged that works in some cases and fix other issues as they're reported.

Reply:
Well, this could possibly be a blocker for merging my PR into main. I'll take this for now, but will have to investigate before releasing this feature.